src/xpk/commands/batch.py

```diff
@@ -46,11 +48,66 @@ def batch(args: Namespace) -> None:
 
 def submit_job(args: Namespace) -> None:
```

Comment:
Probably we also need to have `--skip-localqueue-validation` and `--skip-priority-validation` flags?

Reply:
If localqueue owned by us we don't need to check it. On the other hand user could remove it. So not sure about it to. 

@44past4 @mwysokin @pawloch00 WDYT?

Reply:
For about priority validation the same question. Do we need it?

@44past4 @mwysokin @pawloch00 WDYT?

Reply:
There is only one localqueue defined during cluster creation that is used by xpk. I don't think we should allow user to perform any checks of changes on lq.

Reply:
But lets have @44past4  for final decision

Reply:
The main assumption in XPK is that this should be highly opinionated tool with limited number of options available for users. Therefore in this case I would assume that we own the local queue and priorities and users should not edit those in any way.

Reply:
Thanks!